html5lib/tests/test_sanitizer.py

```diff
@@ -63,25 +63,32 @@ def test_sanitizer():
     for ns, tag_name in sanitizer.allowed_elements:
         if ns != constants.namespaces["html"]:
             continue
-        if tag_name in ['caption', 'col', 'colgroup', 'optgroup', 'option', 'table', 'tbody', 'td',
-                        'tfoot', 'th', 'thead', 'tr', 'select']:
+        if tag_name in ['caption', 'col', 'colgroup', 'optgroup', 'option',
+                        'table', 'tbody', 'td', 'tfoot', 'th', 'thead',
+                        'tr', 'select']:
             continue  # TODO
         if tag_name == 'image':
-            yield (runSanitizerTest, "test_should_allow_%s_tag" % tag_name,
-                   "<img title=\"1\"/>foo &lt;bad&gt;bar&lt;/bad&gt; baz",
-                   "<%s title='1'>foo <bad>bar</bad> baz</%s>" % (tag_name, tag_name))
+            runSanitizerTest("test_should_allow_%s_tag" % tag_name,
```

Comment:
We do want these to be run as individual tests, rather than `test_sanitizer` containing all of them.